validation-test/ModuleInterface/verify_all_overlays.swift

```diff
@@ -0,0 +1,25 @@
+// RUN: %empty-directory(%t)
+// RUN: for x in %platform-sdk-overlay-dir/*.swiftinterface; do [[ $(basename "$x") = Swift.swiftinterface || $(basename "$x") = simd.swiftinterface ]] && continue; %target-swift-frontend "$x" -emit-module -o %t/$(basename "${x/.*}").swiftmodule -disable-objc-attr-requires-foundation-module -enable-resilience -Fsystem %sdk/System/Library/PrivateFrameworks/ || echo -swift-version 4 $(basename "$x") >> %t/failures.txt; done
```

Comment:
This seems like a lot for a RUN line, but I suppose it's alright.

Reply:
:-( Yeah. I could pull it out into some kind of script, I guess? But it's also in SIL/verify_all_overlays.sil.